.gitignore

```diff
@@ -1,3 +1,5 @@
 /node_modules
 /.idea
-package-lock.json
\ No newline at end of file
+package-lock.json
+
+dist/
```

Comment:
I don't know right now if it's recommended or not to push the `dist` directory.

Reply:
generally, we don't push the build folder

Reply:
Okay ✅